controllers/data.controller.js

```diff
@@ -98,18 +98,25 @@ const handleClassicQuestions = async (
   let newScore = updatedScore;
   let newTier = currentTier;
   let response;
-  // demote after 2 wrong answers in a row
-  if (updatedScore < -50) {
-    newScore = 100 + updatedScore; // score is negative so we add it to 100 (ex : 100 + (-50) = 50)
-    // to avoid ranking issues when few users and user should be demoted to D1 when ranked M and above
-    if (tierIndex > TIER_LIST.length - 3) {
-      newTier = "D1";
-    } else {
+  // handle demotion
+  if (tierIndex % 4 === 0) {
```

Comment:
i think here you can check updatedScore too, cant you?

Reply:
I can't because otherwise I can go to the else condition if score = -30 for exemple and that's not a wanted behavior, but I could refactor the else condition